pharmaceuticals_retailers/data/ir_ui_menu.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+    <record id="contacts_doctor_menu_pharma" model="ir.ui.menu">
+        <field name="name">Doctor</field>
+        <field name="sequence">2</field>
+        <field name="parent_id" ref="contacts.menu_contacts" />
+        <field name="action" ref="actions_doctor_partner_pharma" />
+    </record>
+    <record id="contact_doctor_pharma" model="ir.ui.menu">
+        <field name="name">Doctor</field>
+        <field name="web_icon">fa fa-diamond,#f1c40f,#34495e</field>
+    </record>
```

Comment:
I don't see the need for a new app icon... To me, a menu item in Contacts (like the first one above) is more than enough.